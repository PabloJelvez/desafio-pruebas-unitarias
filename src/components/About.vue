<template>
<div class="about text-center" >
    <h1 class="aboutTitle" >Vista About</h1>

</div>

</template>

<script>
export default {
    name: "About",
    data(){
        return{

        }
    }
}

</script>

<style scoped lang="scss">

</style>