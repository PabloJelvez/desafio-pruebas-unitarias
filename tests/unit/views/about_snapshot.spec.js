import { shallowMount } from "@vue/test-utils"
import About from "@/components/About.vue"

describe("Componente About.vue", () => {
    test("ValidaciÃ³n de match con el snapshot", () => {
        const wrapper = shallowMount(About)

        expect(wrapper.html()).toMatchSnapshot()
    });
    test("Validar el titulo de la vista About", () => {
        const wrapper = shallowMount(About)
        const titleAbout = wrapper.find(".aboutTitle")
        expect(titleAbout.text()).toBe("Vista About")
    })
})