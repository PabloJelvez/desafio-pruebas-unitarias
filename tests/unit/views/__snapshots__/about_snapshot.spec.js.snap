// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componente About.vue Validaci√≥n de match con el snapshot 1`] = `
<div class="about text-center">
  <h1 class="aboutTitle">Vista About</h1>
</div>
`;
